packages/components/header/_header.scss

```diff
@@ -500,10 +517,27 @@
   right: 0;
   top: 100%;
   left: 0;
+  margin-left: 0;
+  margin-right: 0;
+
+  @include mq($from: tablet) {
+    margin-left: -16px;
+    margin-right: -16px;
+  }
+
+  @include mq($from: desktop) {
+    margin-left: -32px;
+    margin-right: -32px;
+  }
 
   .nhsuk-header__navigation-link {
     color: $color_nhsuk-blue;
-    padding: 12px nhsuk-spacing(3);
+    padding: 12px 16px;
+
+    @include mq($from: desktop) {
+      padding-left: 32px;
+      padding-right: 32px;
+    }
```

Comment:
Could we have something like this:

```scss
$header-navigation-link-padding-block: 12px;
$header-navigation-link-padding-inline: 16px;
```

Then use these variables elsewhere? That means only needing to change values in one place, and makes it clearer why negative margins are being used elsewhere.